api/src/services/credential-exchange/credential-exchange.class.ts

```diff
@@ -34,6 +36,16 @@ export class CredentialExchange implements ServiceSwaggerAddon {
     this.app = app;
   }
 
+  getSchemaAttrsByID(schema_name: string, schema_version: string): string[] {
+    const schemas = loadJSON("schemas.json") as SchemaDefinition[]
+    const filtered = schemas.filter((s) => s.schema_name === schema_name && s.schema_version === schema_version)
+    let schemaAttributes: string[] = []
+    if (filtered.length > 0 && filtered[0].attributes) {
+      schemaAttributes = filtered[0].attributes
+    }
+    return schemaAttributes
+  }
+
```

Comment:
This would likely be better located in `utils/credential-exchange.ts` as it is a utility method that might be reused elsewhere.

Reply:
change completed